Source/Scene/Globe.js

```diff
@@ -930,10 +930,17 @@ define([
             }
         }
 
-        if (pass.pick && mode === SceneMode.SCENE3D) {
-            // Not actually pickable, but render depth-only so primitives on the backface
-            // of the globe are not picked.
-            commandList.push(this._depthCommand);
+        if (pass.pick) {
```

Comment:
Also, I _kinda_ have the long-term idea that primitives don't even need to create commands for the pick pass unless they want to do an optimization, e.g., if a primitive is a box and an outline, only the box might be in the pick pass.

Instead, the scene could generate pick commands from the color-pass commands by modifying the the shader and commands (and caching them) with some hints from the primitive, e.g., attribute vs. uniform pick ids.